x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.test.tsx

```diff
@@ -117,3 +117,117 @@ describe('EnginesList', () => {
     expect(wrapper.find(LicensingCallout)).toHaveLength(0);
   });
 });
+
+describe('CreateEngineButton', () => {
+  describe('disabled={true}', () => {
+    it('renders a disabled button that shows a popover when focused', () => {
+      const wrapper = mount(<CreateEngineButton disabled />);
```

Comment:
Nice! I didn't know how to test this as the overall component depended on effects to disable the button. 💯